### PR TITLE
Correct related link in json BBE

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
$subject

### CODE REVIEW

**Review by gimantha — DISMISSED** [review]

File: examples/json-type/json_type.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The `json` type can be explained as a union of the simple basic types, `string`,
 
 ## Related links
 - [Access JSON elements](/learn/by-example/access-json-elements/)
-- [Converting to user-defined type](/learn/by-example/converting-to-user-defined-type/)
+- [Converting JSON with langlib functions](/learn/by-example/converting-json-with-langlib-functions/)
```
Comment (gimantha) [inline examples/json-type/json_type.md:13]: Dont we need to say what we convert the jsons to?

Comment (KavinduZoysa) [reply examples/json-type/json_type.md]: Changed

File: examples/index.json [hunk 2]
```diff
@@ -816,8 +816,8 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Converting JSON with langlib functions",
-                "url": "converting-json-with-langlib-functions",
+                "name": "Converting from JSON to user-defined type with langlib functions",
```
Comment (gimantha) [inline examples/index.json:819]:
```suggestion
                "name": "Converting from JSON to user defined type with langlib functions",
```

File: examples/json-type/json_type.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ The `json` type can be explained as a union of the simple basic types, `string`,
 
 ## Related links
 - [Access JSON elements](/learn/by-example/access-json-elements/)
-- [Converting to user-defined type](/learn/by-example/converting-to-user-defined-type/)
+- [Converting from JSON to user-defined type with langlib functions](/learn/by-example/converting-from-json-to-user-defined-type-with-langlib-functions/)
```
Comment (gimantha) [inline examples/json-type/json_type.md:13]:
```suggestion
- [Converting from JSON to user defined type with langlib functions](/learn/by-example/converting-from-json-to-user-defined-type-with-langlib-functions/)
```

**Review by gimantha — APPROVED** [review]

**Pull request merged**